CHANGELOG.md

```diff
@@ -50,6 +50,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * Update Cosmos SDK to 0.42.3
 * Remove deprecated ModuleCdc amino encoding from name module #189
 * Remove deprecated ModuleCdc amino encoding from attribute module #188
+* Metadata Module parsing of base64 public key fixed #225
```

Comment:
This is in the wrong section. It should be 10 lines up under `## Unreleased`.

Reply:
Nice catch!